src/custodian-vs-non-cust-wallet-dodao-academy.yaml

```diff
@@ -334,6 +473,43 @@ steps:
     name: Custodial vs Non-Custodial
     stepItems: []
     uuid: a1366141-af9b-4244-9906-9a9ec299c7f4
+
+  - content: ''
+    name: Evaluation
+    stepItems:
+      - answerKeys:
+          - B
+          - C
+          - D
+        choices:
+          - content: Your funds could get converted into another currency unannounced 
+            key: A
+          - content: The funds are not backed in a 1:1 ratio and are possible to be misused by the exchange
+            key: B
+          - content: They are usually not well regulated
+            key: C
+          - content: The security of your funds is as safe as the security of the exchange
+            key: D
+        content: Why is it risky to use a custodial wallet?
+        type: MultipleChoice
+        uuid: 8a897002-07af-45b0-872c-6c691ab8c01e
+      - answerKeys:
+          - A
+          - C
+        choices:
+          - content: All private keys are stored in one place and hence are more susceptible to being hacked
```

Comment:
What is the difference in A and B, why is B not correct?

May be change the option B